includes/admin/llms.functions.admin.php

```diff
@@ -143,6 +143,20 @@ function llms_get_sales_page_types() {
 	) );
 }
 
+/**
+ * Get an array of available course/membership checkout redirection options
+ * @return   array
+ * @since    [version]
+ * @version  [version]
+ */
+function llms_get_checkout_redirection_types() {
+	return apply_filters( 'llms_checkout_redirection_types', array(
+		'self' => __( '(Default) Return to %s', 'lifterlms' ),
```

Comment:
i think the `sprintf()` found at in includes/admin/views/access-plans/metabox.php at line 20 [link](includes/admin/views/access-plans/metabox.php) should happen within this function.

If this functions is ever used somewhere else then it'll will require additional (non-dry) code to be written every time the function is used.

I *think* this could be rewritten to pull the name from a post type name and fall back to "self" if none is provided. EG:

Something like:

```php
llms_get_checkout_redirection_types( $post_type = null ) {
  $self = __( 'self', ... ); // default.
  if ( $post_type ) {
    $obj = get_post_type_object( $post_type );
    $self = $obj ? $obj->labels->singular_name : $self;
 }
 ...
  'self' => sprintf( __( '(Default) Return to %s', 'lifterlms' ), $self ),
}
```